### ISSUE TITLE
Duplicate route definitions causes 2 `Route` components to be rendered.

### ISSUE THREAD

**Issue by mikemurray** [reactioncommerce/reaction#2319]
## Expected behavior
Routes should be unique

### Actual Behavior
Routes are not unique

![screen shot 2017-05-22 at 10 50 12 am](https://cloud.githubusercontent.com/assets/42217/26321654/80539a1a-3edc-11e7-991e-66bddebbbbe7.png)

### Steps to Reproduce the Behavior

```
    "registry" : [ 
        {
            "route" : "/product/:handle/:variantId?",
            "name" : "product",
            "template" : "productDetailSimple",
            "workflow" : "coreProductWorkflow"
        }, 
        {
            "route" : "/product/:handle/:variantId?",
            "name" : "product",
            "template" : "productDetailSimple",
            "workflow" : "coreProductWorkflow"
        }
    ],
```

### Versions

Node: 7.2.0
NPM: 4.0.5
Meteor Node: 4.8.2
Meteor NPM: 4.5.0
Reaction CLI: 0.8.2
Reaction: 1.2.0
Reaction branch: pr/2303
Docker: 17.03.1-ce

**Comment by aaronjudd** [comment]
@mikemurray @kieckhafer can you reproduce, verify the status here?

**Comment by awadesmoline** [comment]
I created a plugin, with the duplicate routes above but could not get the error. Only one component renders.

**Comment by zenweasel** [comment]
Closed via #2327
